src/dialer.js

```diff
@@ -286,20 +258,64 @@ function addDialer(callSession, withVideo) {
   unmergeButton.hide();
   atxferButton.hide();
   transferButton.hide();
+  reduceVideoButton.hide();
+  expandVideoButton.hide();
+  expandLocalVideo.hide();
+  reduceLocalVideo.hide();
+  timer.show();
+
+  contact.html(label);
 
   // Videos
-  const videoContainer = $('.videos', newDialer);
+  const videoContainer = $('.videos', newScene);
   if (withVideo) {
     videoContainer.show();
+    videoContainer.addClass('background-videocall');
+    $('.buttons').addClass('buttons-video');
+    reduceVideoButton.show();
+    reduceLocalVideo.show();
+
+    // Reduce & Expand Remote video screen
+    
+    reduceVideoButton.on('click', e => {
+      e.preventDefault;
+      $('.remote video').addClass('reduce-video');
+      reduceVideoButton.hide();
+      expandVideoButton.show();
+    })
+
+    expandVideoButton.on('click', e => {
+      e.preventDefault;
+      $('.remote video').removeClass('reduce-video');
+      expandVideoButton.hide();
+      reduceVideoButton.show();
+    })
+
+     // Reduce & Expand Local video screen
+
+     reduceLocalVideo.on('click', e => {
+      e.preventDefault;
+      $('.local video').addClass('reduce-video');
+      reduceLocalVideo.hide();
+      expandLocalVideo.show();
+    })
+
+    expandLocalVideo.on('click', e => {
+      e.preventDefault;
+      $('.local video').removeClass('reduce-video');
+      expandLocalVideo.hide();
+      reduceLocalVideo.show();
+    })
+
 
     // Local video
     const localStream = Wazo.Phone.getLocalVideoStream(callSession);
-    const localVideo = $('video.local', newDialer)[0];
+    const localVideo = $('.local video', newScene)[0];
     localVideo.srcObject = localStream;
     localVideo.play();
 
     // Remote video
-    const $remoteVideo = $('video.remote', newDialer);
+    const $remoteVideo = $('.remote video', newScene);
```

Comment:
It's a good practice to prefix jquery elements with `$` but it should be applied to all elements or none :)